pkg/scheduler/extender/extender.go

```diff
@@ -447,20 +446,20 @@ func (e *Extender) createReservation(ctx context.Context, namespace string, name
 	return nil
 }
 
-func (e *Extender) prepareListOfRequestedNodes(nodes []coreV1.Node) ([]string, error) {
-	requestedNodes := make([]string, len(nodes))
+func (e *Extender) prepareListOfRequestedNodes(nodes []coreV1.Node) []string {
+	requestedNodes := []string{}
 
-	for i, node := range nodes {
-		node := node
-		nodeID, err := annotations.GetNodeID(&node, e.annotationKey, e.featureChecker)
+	for _, node := range nodes {
+		n := node
+		nodeID, err := annotations.GetNodeID(&n, e.annotationKey, e.featureChecker)
 		if err != nil {
-			e.logger.Errorf("failed to get NodeID: %s", err)
-			return nil, err
+			e.logger.Errorf("node:%s cant get NodeID error: %s", n.Name, err)
```

Comment:
We need to check node label here. When node has label which was set during CSI installation need to return an error, otherwise just skip such node without even logging error.

Reply:
Could you explain a little, I don't get.

Reply:
When you deploy CSI on subset of nodes using labels you will always get this error for nodes which don't have them. It sounds logical to let extender know which labels are used and don't event try to get node UUID for them.

Reply:
👍